app/services/ClaimService.scala

```diff
@@ -69,8 +69,8 @@ class ClaimService @Inject() (
     maybeAmendRequest match {
       case Some(value) =>
         connector.submitAmendClaim(value, correlationId(hc)).map { clientClaimResponse =>
-          if (clientClaimResponse.result.map(_.caseId).nonEmpty)
-            clientClaimResponse.result.map(_.caseId).get
+          if (clientClaimResponse.caseId.nonEmpty)
+            clientClaimResponse.caseId.get
```

Comment:
I know this was already like this, but can this be refactored to get rid of the "if" and "get" statements?

Reply:
on it